docs/src/main/asciidoc/extension-authors-guide.adoc

```diff
@@ -1233,6 +1233,130 @@ HealthBuildItem addHealthCheck(AgroalBuildTimeConfig agroalBuildTimeConfig) {
 }
 ----
 
+=== Customizing Json handling from an extension
```

Comment:
Let's use `JSON` as is done in the `rest-json` guide.

Reply:
done